opensoundscape/audio.py

```diff
@@ -1299,15 +1297,16 @@ def estimate_delay(
         reference_audio: audio object containing the reference signal.
         max_delay: maximum time delay to consider, in seconds. Must be less than the duration of the primary audio.
             (see `opensoundscape.signal_processing.tdoa`)
-        bandpass_range: if None, no bandpass filter is performed
-            otherwise [low_f,high_f]
-        bandpass_order: order of Butterworth bandpass filter
+        frequency_range: tuple of (low_f, high_f) frequencies in Hz to use in the generalized cross correlation.
+            If None, all frequencies are kept.
+            First or second value can be None if no lower or upper limit is desired
+            Note: retaining high frequencies near the Nyquist frequency sometimes results
+                in spurious cross correlation values at 0 or at the beginning/end of the signal
+                when using 'phat' and 'scot' methods.
```

Comment:
No changes requested. I just want to understand this.

Reply:
sorry it has been too long since I wrote that to remember the context or provide an example, but either way I don't understand why it occurs